integration-tests/cucumber-tests-platform-distributionautomation/src/test/java/org/opensmartgridplatform/cucumber/platform/distributionautomation/PlatformDistributionAutomationDefaults.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.cucumber.platform.distributionautomation;
+
+public class PlatformDistributionAutomationDefaults {
+
+    public static final Integer COMMON_ADDRESS = 1511;
```

Comment:
We currently have a mix of "prefixed" and "non-prefixed" defaults and keys. Because it's often not obvious from which class a default or key exactly comes from, I would opt for non-prefixed versions here and use the "classname.field" syntax.